dpnp/dpnp_iface_linearalgebra.py

```diff
@@ -231,10 +290,11 @@ def vdot(*args, **kwargs):
     """
     Return the dot product of two vectors.
 
+    For full documentation refer to :obj:`numpy.vdot`.
+
     See Also
-    --------
-    :obj:`numpy.vdot`
+    ---------
```

Comment:
```suggestion
    --------
```

Maybe we will mention that the function works the same as :obj:`dpnp.dot` in `Notes` block. What do you think about that?